include/deal.II/dofs/dof_levels.h

```diff
@@ -38,14 +38,8 @@ namespace internal
      * Structure for storing degree of freedom information for cells,
      * organized by levels.
      *
-     * We store are cached values for the DoF indices on each cell
-     * in#cell_dof_indices_cache, since this is a frequently requested
-     * operation. The values are set by
-     * DoFCellAccessor::update_cell_dof_indices_cache and are used by
-     * DoFCellAccessor::get_dof_indices.
-     *
      * Note that vertices are separate from, and in fact have nothing to do
-     * with cells. The indices of degrees of freedom located on vertices
+     * with, cells. The indices of degrees of freedom located on vertices
```

Comment:
Is this new comma here correct? I would have put "in fact" between commas.

Reply:
I guess it depends on how you read the sentence. I can understand your reasoning. Mine was that if you put the comma before "and", one needs to make the subordinate clause all the way to its end, i.e., after "with". Either way, this sentence feels a bit too wrinkled (German?), so let me rephrase this a bit more.

Reply:
How about the new version?